modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase

```diff
@@ -488,6 +482,39 @@ definition {
 		}
 	}
 
+	@description = "Verify that the value of Total MDF Requested Amount* will be according to the Total cost and Claim percent"
+	@priority = 5
+	test RequestedValueIsEqualsTo50PercentOfTotalCost {
+		task ("Given a new MDF request") {
+			PRMMarketing.gotoAddNewMDF();
+
+			PRMMDFRequest.fillGoalsPage(
+				accountName = "Company Name 1",
+				additionalOptions = "6.x to DXP Migration/Upgrade,Migration from competitor platform",
+				campaignDescription = "Campaign Description",
+				campaignName = "Campaing Name",
+				countryName = "Mexico",
+				goalsOptions = "Lead generation",
+				targetMarketOptions = "Education,Energy",
+				targetRolesOptions = "Administrator");
+
+			Button.click(button = "Continue");
+		}
+
+		task ("When I create a new activity") {
+			PRMMDFRequest.addNewActivity();
+		}
+
+		task ("And When I add a new expense") {
+			PRMMDFRequest.fillRequestAmountField(
+				budgetList = "Direct Mail-100000,Room Rental-3000");
```

Comment:
Hi @mateussandes23, here isn't necessary to add two budgets. Could you please add just one? Thanks!